assets/css/styles.css

```
@@ -60,6 +65,8 @@ body {
   font-size: var(--normal-font-size);
   font-weight: 500;
   overflow-x: hidden;
+  /* Remove this to get the original white background */
+  background-color: var(--new-black-bg);
```

```suggestion
  /* background-color: var(--new-black-bg); */
```

This need to be removed if you want to keep the white background for now.

---

Yeah, thank you!